calculator.c

```diff
@@ -0,0 +1,52 @@
+/* Yael Kelmer.
```

Comment:
Hi Yael, your code compiled, but you didn't follow instructions correctly! You didn't to be able to accept float numbers as input!